action_server/src/action_server/actions/find.py

```diff
@@ -5,7 +5,7 @@
 import rospy
 import math
 
-import robot_smach_states as states
+import robot_smach_states as rss
```

Comment:
It might be nicer to explicitly import the used states. Most of tue_robocup still uses `import ... as states`

Reply:
The imports have changed, so you need to include the submodule name as well, and to keep it short, rss is shorter than states.